csp-mixin/panels.libsonnet

```diff
@@ -899,5 +899,487 @@ local commonlib = import 'common-lib/common/main.libsonnet';
       + g.panel.timeSeries.fieldConfig.defaults.custom.stacking.withMode('normal')
       + this.signals.azurevirtualnetwork.tcpPacketsForwarded.asPanelMixin()
       + this.signals.azurevirtualnetwork.udpPacketsForwarded.asPanelMixin(),
+
+    // Gcp Compute Engine
+
+    gce_instance_count:
+      this.signals.gcpceOverview.instanceCount.asStat()
+      + commonlib.panels.generic.stat.base.stylize(),
+    gce_system_problem_count:
+      this.signals.gcpceOverview.systemProblemCount.asStat()
+      + commonlib.panels.generic.stat.base.stylize(),
+    gce_top5_cpu_utilization:
+      this.signals.gcpceOverview.top5CpuUtilization.asTable(format='table')
+      + commonlib.panels.generic.table.base.stylize()
```

Comment:
Whenever you use the `as<panel type>` functions on signals, the appropriate styling is already applied, so these lines are superfluous.